pandas/core/window/rolling.py

```diff
@@ -558,6 +558,10 @@ def __init__(
         if _grouper is None:
             raise ValueError("Must pass a Grouper object.")
         self._grouper = _grouper
+        # GH 32262: It's convention to keep the grouping column in
```

Comment:
is the difference in conventions just a historical thing?  might we want to make these match eventually?

Reply:
Seems to be historical https://github.com/pandas-dev/pandas/pull/32332#pullrequestreview-367673502

Would be great to have groupby apply match but might take come convincing